crates/uv/tests/pip_sync.rs

```diff
@@ -111,12 +111,13 @@ fn missing_venv() -> Result<()> {
     fs::remove_dir_all(&context.venv)?;
 
     uv_snapshot!(context.filters(), sync_without_exclude_newer(&context).arg("requirements.txt"), @r###"
-    success: false
-    exit_code: 2
+    success: true
+    exit_code: 0
     ----- stdout -----
 
     ----- stderr -----
-    error: No Python interpreters found in virtual environments
+    Resolved 1 package in [TIME]
+    Audited 1 package in [TIME]
```

Comment:
Interesting regression... will investigate this.

Reply:
These tests are not properly isolated from managed toolchains because they aren't following our `TestContext` pattern :sigh: